### DISCUSSION TITLE
Cmdline popup appears in every character when I have error in my code

### DISCUSSION THREAD

**Discussion by mortymacs** [folke/noice.nvim#363] [Q&A]
When I open up a file containing an error, I move the cursor on that line, and the cmdline popup will appear on every cursor movement.

This is what I see:

https://user-images.githubusercontent.com/1261448/219899447-1699cead-385b-4748-9b6d-bc3a82ed4b74.mp4

Also, based on the document, I tried to disable the message inside the Noice LSP config:
```lua
require("noice").setup({
  lsp = {
    ...
    message = {
      enabled = false,
    },
  ...
})
```
But it shows the error that I should disable either in LSP or Noice, which I just already did in Noice:
![image](https://user-images.githubusercontent.com/1261448/219899471-2e5cc040-8204-46bd-9187-ade70aca0a2f.png)

This is my Noice config:
```lua
require("noice").setup({
  lsp = {
    signature = {
      enabled = false,
    },
    hover = {
      enabled = false,
    },
  },
  messages = {
    enabled = false,
  },
  popupmenu = {
    enabled = false,
  },
  cmdline = {
    format = {
      cmdline = { icon = ">" },
      lua = { icon = ">" },
    },
  },
  presets = {
    bottom_search = true,
    command_palette = true,
    long_message_to_split = true,
    inc_rename = false,
    lsp_doc_border = false,
  },
  views = {
    cmdline_popup = {
      size = {
        width = 100,
        height = "auto",
      },
      border = {
        style = "single",
        padding = { 0, 0, },
      },
    },
  },
})
```

**Comment by folke** [comment]
You seem to have an autocmd that does `:echom a: message`. That's not noice. Check your config and your plugins

**Reply by mortymacs** [reply]
I ran grep and I didn't find anything, I also disabled all the plugins but Noice. Still, it shows that error.

https://user-images.githubusercontent.com/1261448/219938315-fecfd3a5-2e91-4579-ba0e-d158f68b05c8.mp4

**Reply by mortymacs** [reply]
It seems, when LSP is enabled, it will trigger this behavior automatically.

**Reply by folke** [reply]
Still the same, not Noice. You see the cmdline, because whatever is causing that message is not using `silent`

**Reply by mortymacs** [reply]
Finally, I found the conflict. It was the conflict with the ALE package. So, I removed ALE, and now everything works perfectly.
